extension/lib/config.coffee

```diff
@@ -5,6 +5,8 @@ messageManager = require('./message-manager')
 utils = require('./utils')
 
 {OS} = Components.utils.import('resource://gre/modules/osfile.jsm', {})
+ffRequire = Cu.import('resource://devtools/shared/Loader.jsm').require
+Services = ffRequire("Services")
```

Comment:
Can you fix the global `Services` injection in bootstrap.coffee instead?

Reply:
I tried, and failed :|

If you'd tell me what I'd need to put where, than I'll do it, of course. Otherwise, I'd leave it, since there aren't many people interested in this code anymore anyways

Reply:
Here's the code that loads `Services`:

https://github.com/akhodakivskiy/VimFx/blob/687923bd151b0fe1fad9056511cbdd509e7b6bf4/extension/bootstrap.coffee#L23

That must work in bootstrap.coffee itself, since the usages of `Services` in that file itself seems to work. But it seems all other files are no longer affected by it. You could try changing to `Cu.import('resource://gre/modules/Services.jsm')
Cu.import('resource://gre/modules/Console.jsm', global)` and see if it makes a difference. Otherwise you should be able to add `Services` to the `scope` here:

https://github.com/akhodakivskiy/VimFx/blob/687923bd151b0fe1fad9056511cbdd509e7b6bf4/extension/bootstrap.coffee#L49-L56

Btw, does `console` work everywhere? It is loaded the same way as `Services`. If not, we should do the same thing for `console`.

Reply:
console works, because we aren't accessing anything other than the `console` object, which doesn't need explicit importing.

adding `global` did nothing, mdn says, this is the default if no 2nd param given.

I've changed bootstrap.coffee:52 from `Cc, Ci, Cu` -> `Cc, Ci, Cu, Services`, that seemed to do the trick. I'll push this after testing in ff64, wf56, wf68

Reply:
> the console object, which doesn't need explicit importing.

It used to. ¯\\\_(ツ)\_/¯